test/examples/planar.js

```diff
@@ -14,4 +14,36 @@ describe('planar', () => {
         assert.ok(result);
         await page.close();
     });
+    it('should get picking position from depth', async function _() {
+        const page = await browser.newPage();
+
+        await page.setViewport({ width: 400, height: 300 });
+        await page.goto(`http://localhost:${itownsPort}/examples/planar_vector.html`);
+        await page.waitFor('#viewerDiv > canvas');
+        await exampleCanRenderTest(page, this.test.fullTitle());
+
+        const length = 1500;
+
+        // get range with depth buffer and altitude
+        const result = await page.evaluate((l) => {
+            const lookat = extent.center().xyz();
+
+            view.camera.camera3D.position.copy(lookat);
+            view.camera.camera3D.position.z = l;
+            view.camera.camera3D.lookAt(lookat);
+            view.notifyChange(view.camera.camera3D, true);
+            const depthMethod = view
```

Comment:
But here you should wait for all textures to be loaded - because you're using them in DEMUtils below.

Reply:
and `depthMethod` is `distanceUsingDepthMethod`